romtools/vector_space/utils/scaler.py

```diff
@@ -214,7 +215,7 @@ def initialize_scalings(self, data_tensor: np.ndarray) -> None:
         for i in range(n_var):
             ith_var = data_tensor[i]
             if self.__scaling_type == 'max_abs':
-                var_scale = np.max(abs(ith_var))
+                var_scale = pla.max(abs(ith_var))
             elif self.__scaling_type == 'mean_abs':
                 var_scale = np.mean(abs(ith_var))
```

Comment:
why are we not using the pla for the mean?

Reply:
No reason--I'll add that in ASAP